README.md

```diff
@@ -109,3 +109,90 @@ class Version20130403165433 extends AbstractMigration
     }
 }
 ```
+
+### Accessing Zend Db Adapter In Migration Class
+
+By implementing the `Zend\Db\Adapter\AdapterAwareInterface` in your migration class you get access to the
+Db Adapter configured for the migration.
+
+```php
+<?php
+
+namespace ZfSimpleMigrations\Migrations;
+
+use Zend\Db\Adapter\AdapterAwareInterface;
+use Zend\Db\Adapter\AdapterAwareTrait;
+use Zend\Db\Sql\Ddl\Column\Integer;
+use Zend\Db\Sql\Ddl\Column\Varchar;
+use Zend\Db\Sql\Ddl\Constraint\PrimaryKey;
+use Zend\Db\Sql\Ddl\CreateTable;
+use Zend\Db\Sql\Ddl\DropTable;
+use ZfSimpleMigrations\Library\AbstractMigration;
+use Zend\Db\Metadata\MetadataInterface;
+
+class Version20150524162247 extends AbstractMigration implements AdapterAwareInterface
+{
+    use AdapterAwareTrait;
```

Comment:
My projects run on top of php 5.5 and I use traits as well, but when I added `ServiceLocatorAwareInterface ` support to the module I tried to make it php 5.3 compatible. As php53 is currently outdated, please update _Accessing ServiceLocator_ section of the doc to use trait instead of property/getter/setter. Thanks.